gnssanalysis/gn_io/sp3.py

```diff
@@ -39,19 +39,31 @@
 SP3_CLOCK_NODATA_STRING = " 999999.999999"  # Not used for reading, as fractional components are optional
 SP3_CLOCK_NODATA_NUMERIC = 999999
 SP3_POS_NODATA_STRING = "      0.000000"
+SP3_POS_NODATA_NUMERIC = 0
 SP3_CLOCK_STD_NODATA = -1000
 SP3_POS_STD_NODATA = -100
 
 
+def sp3_pos_nodata_to_nan(sp3_df):
```

Comment:
It's a long term clean-up project, but we're hoping to slowly move all python code to follow a documentation standard. In our case, we've chosen one-line sphinx style: https://github.com/NilsJPWerner/autoDocstring/blob/HEAD/docs/one-line-sphinx.md

There's an example of this in the compare_clk function: https://github.com/GeoscienceAustralia/gnssanalysis/blob/2297ba76bb8833988245351db8779d49391d0ff0/gnssanalysis/gn_diffaux.py#L307

Ideally all new functions follow this format and in future we'll work towards going through the backlog and changing all the old functions to the new doc standard as well

I think the main thing for your function `sp3_pos_nodata_to_nan()` is to include type hints and an explanation of the input / outputs in the Docstring